src/services/models/user-profile.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserProfile {
+    alias: string;
+    email: string;
+    avatar: string;
+    since: Date;
```

Comment:
Firebase has a special date type https://firebase.google.com/docs/reference/js/firebase.firestore.Timestamp

```typescript
firebase.firestore.FieldValue.serverTimestamp()
```

Reply:
that's for firestore.... fireauth seems to define it as

```
  interface UserMetadata {
    creationTime?: string;
    lastSignInTime?: string;
  }
```

Reply:
Oh yes for Firestore is what I was thinking